src/TestcafeGherkinBootstrapper.js

```diff
@@ -100,10 +100,16 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
 
   _resolveAndRunStepDefinition(testController, step) {
     for (const stepDefinition of this.stepDefinitions) {
-      const match = stepDefinition.pattern.exec(step.text);
-
-      if (match) {
-        return this._runStep(stepDefinition.code, testController, match.slice(1));
+      if (typeof stepDefinition.pattern === 'string') {
+        let isMatched = stepDefinition.pattern.toLowerCase().trim() === step.text.toLowerCase().trim();
```

Comment:
I don't really like case insensitive matching. I think matching should be case sensitive (just as it is by default for regular expressions).
Plus `trim`ing does not really make sense since the string will be trimmed by the Gherkin parser.